druid/examples/markdown_preview.rs

```diff
@@ -14,27 +14,32 @@
 
 //! An example of live markdown preview
 
-use std::ops::Range;
-
 use pulldown_cmark::{Event as ParseEvent, Parser, Tag};
 
-use druid::text::{Attribute, AttributeSpans, RichText};
+use druid::text::{AttributesAdder, RichText, RichTextBuilder};
 use druid::widget::prelude::*;
 use druid::widget::{Controller, LineBreaking, RawLabel, Scroll, Split, TextBox};
 use druid::{
-    AppLauncher, Color, Data, FontFamily, FontStyle, FontWeight, Lens, LocalizedString, MenuDesc,
-    Widget, WidgetExt, WindowDesc,
+    AppDelegate, AppLauncher, Color, Command, Data, DelegateCtx, FontFamily, FontStyle, FontWeight,
+    Handled, Lens, LocalizedString, MenuDesc, Selector, Target, Widget, WidgetExt, WindowDesc,
 };
 
 const WINDOW_TITLE: LocalizedString<AppState> = LocalizedString::new("Minimal Markdown");
 
 const TEXT: &str = "*Hello* ***world***! This is a `TextBox` where you can \
-                   use limited markdown notation, which is reflected in the \
-                   **styling** of the `Label` on the left.";
+		    use limited markdown notation, which is reflected in the \
+		    **styling** of the `Label` on the left.
+
+A very good place to ask questions and discuss development work is our [Zulip chat instance],\
+in the #druid-help and #druid channels, respectively.
+
+[Zulip chat instance]: https://xi.zulipchat.com
+";
```

Comment:
formatting nit, I'm sorry, I can't help myself 😭 

```suggestion
		    **styling** of the `Label` on the left.

		    If you're curious about Druid, a good place to ask questions \
		    and discuss development work is our [Zulip chat instance], \
		    in the #druid-help and #druid channels, respectively.\n\n\n\
		    
		    [Zulip chat instance]: https://xi.zulipchat.com";
```